extensions/amp-form/amp-form.md

```diff
@@ -129,9 +129,8 @@ This attribute is required for `method=POST`, and is optional for `method=GET`.
 
 The value for `action-xhr` can be the same or a different endpoint than `action` and has the same `action` requirements above.
 
-To learn about redirecting the user after successfully submitting the form, see the [Redirecting after a submission](#redirecting-after-a-submission) section below.
-
 [filter formats="websites, stories, ads"]
```

Comment:
It looks like the "Redirecting after a submission" section is only available to web and ad formats. We should either remove "stories" here or add "stories" there, and I _think_ redirects would be disallowed in stories? @ampproject/wg-stories

Reply:
Good point. Stories don't support amp-form at all. I removed `stories` in my updated PR https://github.com/ampproject/amphtml/pull/26978.